### DISCUSSION TITLE
Updating Sunbird Collection Editor Library to Angular 16 and updating peer dependencies

### DISCUSSION THREAD

**Discussion by abhishekpnt** [Sunbird-Knowlg/Community#196] [Bugs]
@pallakartheekreddy 

@project-sunbird/sunbird-collection-editor is using non-ivy library ng2-semantic-ui-v9 in peer dependencies in package.json which is causing build issues in portal.

Please update this library with angular 16 version using [npm: @project-sunbird/ng2-semantic-ui](https://www.npmjs.com/package/@project-sunbird/ng2-semantic-ui)with version 8.0.2 instead of ng-semantic-ui-v9 and provide the new npm package

Refer: https://project-sunbird.atlassian.net/browse/ED-3731
https://project-sunbird.atlassian.net/browse/ED-3836

CC: @vinukumar-vs @rajeevsathish @mohitga

**Comment by pallakartheekreddy** [comment]
@abhishekpnt Collection editor also having web component right? Why can't you use that? Why do you need only NPM package?

**Comment by abhishekpnt** [comment]
@pallakartheekreddy if we use collection editor web compontent, we are getting the errors mentioned in the following discussion thread.
https://github.com/orgs/Sunbird-Knowlg/discussions/185

So ca you please provide us the latest npm package

CC: @vinukumar-vs @rajeevsathish @mohitga

**Reply by pallakartheekreddy** [reply]
@abhishekpnt As discussed, we will provide a new NPM package for this fix. 
Please use `@project-sunbird/sunbird-collection-editor@6.1.0`
